.gitignore

```diff
@@ -30,5 +30,6 @@ project/target
 # metals
 .metals/
 project/.bloop/
+.bloop/
```

Comment:
I'm not against this, but the good practice here is to add it to your personal `$HOME/.gitignore` file, so you don't need to add this tool for each project you use it. :wink:

Reply:
Should I delete it?

Reply:
No, no worries. It was JFYI, since I have detected it is widely done be most of us :smile: